Gemfile

```diff
@@ -85,4 +85,6 @@ gem 'cancancan'
 
```

Comment:
- [ ] On your `gemfile`, some gems are not being used (seems the case with `cancancan `for instance).  Kindly consider deleting the gems that are not being used.

That way, the dependencies will be consistent.